wagtail_localize/models.py

```diff
@@ -876,6 +879,75 @@ def schema_out_of_date(self):
         )
         return self.schema_version != current_schema_version
 
+    def sync_view_restrictions(self, original, translation_page):
+        """
+        Synchronizes view restriction object for the translated page
+
+        Args:
+            original (Page|Snippet): The original instance.
+            translation_page (Page|Snippet): The translated instance.
+        """
+        if not isinstance(original, Page) or not isinstance(translation_page, Page):
+            return
+
+        if original.view_restrictions.exists():
+            original_restriction = original.view_restrictions.first()
+            if not translation_page.view_restrictions.exists():
+                view_restriction, child_object_map = _copy(
```

Comment:
Minor style point but `child_object_map` is an used variable here

Normally I'd suggest

```py
view_restriction, _ = _copy(...
```

but we've done `from django.utils.translation import gettext as _` in this file, so probably not that :)

If we wanted to avoid the unused var we could do 


```py
view_restriction = _copy(
    original_restriction,
    exclude_fields=["id"],
    update_attrs={"page": translation_page}
)[0]
```

I think I'll just flag it and leave you to decide what you think is most readable though

Reply:
definitely not `_` because of the `gettext` import

we have a number of other places where this is happening, so I follow it. the `[0]` variant will work, but it is not the most readabale so I think we can live with the unused var